### DISCUSSION TITLE
Pending scheduled Jobs

### DISCUSSION THREAD

**Discussion by theurichde** [renovatebot/renovate#22540] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

We have a bunch of repositories that have a pending job. The last run was for the majority 2 days ago, for some 3 days ago. The schedule is "every weekday". 

I'm wonder if we ran into some rate limit on your side?

Can we do something to fix those pending jobs and give them a little kick?

Some examples with job id:
* `01886f23-ff00-7753-b719-7c641048c3bb`
* `01886f23-f9d8-7f52-a509-90831b8e9411`

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Are these repos in onboarded, or still have the onboarding PR open?

**Reply by mgerlach** [reply]
All onboarded by automatically adding renovate.json to the main branch (which auto-closed the onboarding PR)

**Comment by rarkins** [comment, marked as answer]
Hi, we've found a bug where commits to onboarding branches are being ignored, meaning Renovate does not immediately run again. In addition, daily jobs, which are used to check onboarding repos, were not getting prioritized enough. We are adjusting resources so that daily jobs should get executed once per day as intended, and the commit webhook bug should be fixed on Sunday this week.

**Reply by mgerlach** [reply]
Awesome, PRs are already being opened! Thanks for the quick fix!
